Test/Unit/Helper/ChargedCurrencyTest.php

```diff
@@ -257,32 +263,61 @@ protected function setUp(): void
             ]
         );
 
-
-        $creditMemo = $this->getMockBuilder(Order\Creditmemo::class)
+        $this->creditMemo = $this->getMockBuilder(Order\Creditmemo::class)
             ->disableOriginalConstructor()
             ->setMethods([
-                'getInvoice'
+                'getBaseGrandTotal',
+                'getBaseCurrencyCode',
+                'getBaseTaxAmount',
+                'getBaseAdjustment',
+                'getBaseShippingAmount',
+                'getBaseShippingTaxAmount',
+                'getGrandTotal',
+                'getOrderCurrencyCode',
+                'getTaxAmount',
+                'getAdjustment',
+                'getShippingAmount',
+                'getShippingTaxAmount',
+                'getInvoice',
+                'getOrder'
             ])
             ->getMock();
-        $creditMemo->method('getInvoice')->willReturn($this->invoice);
+        $this->mockMethods($this->creditMemo,
+            [
+                'getBaseGrandTotal' => self::AMOUNT_CURRENCY['base']['amount'],
+                'getBaseCurrencyCode' => self::AMOUNT_CURRENCY['base']['currencyCode'],
+                'getBaseTaxAmount' => self::AMOUNT_CURRENCY['base']['taxAmount'],
+                'getBaseAdjustment' => self::AMOUNT_CURRENCY['base']['amount'],
+                'getBaseShippingAmount' => self::AMOUNT_CURRENCY['base']['amount'],
+                'getBaseShippingTaxAmount' => self::AMOUNT_CURRENCY['base']['taxAmount'],
+                'getGrandTotal' => self::AMOUNT_CURRENCY['display']['amount'],
+                'getOrderCurrencyCode' => self::AMOUNT_CURRENCY['display']['currencyCode'],
+                'getTaxAmount' => self::AMOUNT_CURRENCY['display']['taxAmount'],
+                'getAdjustment' => self::AMOUNT_CURRENCY['display']['amount'],
+                'getShippingAmount' => self::AMOUNT_CURRENCY['display']['amount'],
+                'getShippingTaxAmount' => self::AMOUNT_CURRENCY['display']['taxAmount'],
+                'getInvoice' => $this->invoice,
+                'getOrder' => $this->order
+            ]
+        );
 
         $this->creditMemoItem = $this->getMockBuilder(Item::class)
             ->disableOriginalConstructor()
             ->setMethods([
                 'getBasePrice',
                 'getPrice',
-                'getInvoice',
                 'getCreditMemo',
+                'getOrder',
                 'getBaseTaxAmount',
                 'getTaxAmount',
-                'getQty'
+                'getQty',
```

Comment:
let's leave out the trailing commas
```suggestion
                'getQty'
```